doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1259,30 +1259,29 @@ Managing the local context
 
 .. tacn:: clear {? {? - } {+ @ident } }
 
-   clear
-   clear ident ...
-   clear - ident ...
-
    Erases hypotheses from the context of the current goal, provided that the
    remaining hypotheses and the goal don't depend directly or indirectly on the
    erased hypotheses.
    That means the hypotheses will no longer appear in the context and therefore
    can't be used in subsequent proof steps.
 
    :n:`clear`
-     All hypotheses are erased.  This may leave the context unchanged; this form never fails.
+     All hypotheses are erased.  This may leave the context unchanged; this form
+     never fails.
 
    :n:`clear {+ @ident }`
      Erases the named hypotheses.
 
-   :n:`clear - {+ @ident }`
-     Erases all hypotheses that are not named by(ass) the :n:`@ident`\s.
+      .. exn:: @ident is used in the conclusion.
+         :undocumented:
 
-   .. exn:: @ident is used in the conclusion.
-      :undocumented:
+      .. exn:: @ident is used in the hypothesis @ident.
+         :undocumented:
 
-   .. exn:: @ident is used in the hypothesis @ident.
-      :undocumented:
+   :n:`clear - {+ @ident }`
+     Erases all hypotheses that are not named by the :n:`@ident`\s.  This may
+     leave the context unchanged; this form never fails if the
+     :n:`@ident`\s name hypotheses in the context1299.
```

Comment:
Looks like some garbage appeared on this line. BTW, I think we should really clarify one way or another that this does not erase hypotheses that depend on named hypotheses.